deploy/gcloud/README.md

```diff
@@ -0,0 +1,89 @@
+# AMP Packager Google Cloud Deployment
+
+This page shows you how to deploy a [Kubernetes](https://kubernetes.io/) cluster of
+ [AMP Packager](https://github.com/ampproject/amppackager#amp-packager) in a Google Cloud environment:
+
+  1. Setup your production or development environment in preparation for the cloud deployment.
+  2. Modify the setup.sh script to enter all information relevant to your cloud
+     deployment.
+  3. Run the gcloud_up.sh to initiate the deployment.
+  4. Run the gcloud_down.sh to tear down the deployment, if desired.
+
+## Before you begin
+
+These instructions are mostly lifted from the [Google Kubernetes Engine tutorial](https://cloud.google.com/kubernetes-engine/docs/tutorials/hello-app).  You may want to consult that page for further reference.  The following items are required before you can begin your deployment:
+
+  1. Visit the [Kubernetes Engine page](https://pantheon.corp.google.com/projectselector2/kubernetes?_ga=2.129894223.2142802049.1581352297-128100078.1565897713) in the Google Cloud Console.
```

Comment:
update to https://console.cloud.google.com/

Reply:
Done, but also added testing section to README, kindly take another look.